tests/test_wcs.py

```diff
@@ -2047,9 +2052,14 @@ def test_astropywcs():
     # These all work, but it is quite slow, so only test one of them for the regular unit tests.
```

Comment:
"only test one of them" -> "only test a few of them"

And similar comment below.